CodeStarterCamp_Week1/Step3.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Step3.swift
+//  CodeStarterCamp_Week1
+//
+//  Created by Jeong Min Lee on 2/13/24.
+//
+
+import Foundation
+
+func drawPepero(bodyLength length: Int, peperoBody body: String, topping: String, stickLength sticklength : Int) {
+    print ("""
+            <정보>
+            길이: \(length)
+            몸통: \(body)
+            토핑: \(topping)
+            막대길이: \(sticklength)
+           """)
+    drawPeperoBody(length: length, body: body, topping: topping)
+    drawPeperoStick(sticklength: length, body: body)
+}
+
+func drawPeperoBody(length: Int, body: String, topping: String) {
+    for _ in 1...length {
+        print(" ",topping, body, topping, separator: "")
+    }
+}
+
+func drawPeperoStick(sticklength: Int,body: String) {
+    for _ in 1...sticklength {
+        let totalLength = body.count + 2
+        let bar = String(repeating: " ", count: totalLength / 2 - 1)
```

Comment:
```suggestion
    let totalLength = body.count + 2
    let bar = String(repeating: " ", count: totalLength / 2 - 1)
    for _ in 1...sticklength {
```

for문을 돌며 매번 새로 할당하게 되는 상수입니다. 반복문 내부에서 선언할 필요가 없을 것 같습니다.